test/test_files/tinysnb/projection/single_label.test

```diff
@@ -667,6 +667,11 @@ Runtime exception: Found duplicate key: {a: 5, b: 3} in map.
 ---- error
 Runtime exception: Found duplicate key: 0 in map.
 
+-LOG MapWithNullKey
+-STATEMENT RETURN map([cast(5, 'int128'), NULL, cast(5, 'int128')], [20, 34, 50]);
+---- error
+Runtime exception: Null value key is not allowed in map.
```

Comment:
Are we sure about this? Thought null key could be supported.

Reply:
```
D SELECT MAP {'key1': 5, null: 43};
Error: Invalid Input Error: Map keys can not be NULL
```
Duckdb doesn't support null as keys